imports/plugins/included/sitemap-generator/server/lib/generateSitemaps.js

```diff
@@ -0,0 +1,144 @@
+import { Meteor } from "meteor/meteor";
+import { check, Match } from "meteor/check";
+import Hooks from "@reactioncommerce/hooks";
+import Reaction from "/imports/plugins/core/core/server/Reaction";
```

Comment:
This should be imported from `/server/api`

Reply:
@zenweasel Release 1.14 changes that. It looks like all references to `/server/api` have been moved to `/imports/plugins/core/core/server/Reaction` it looks like.

Reply:
as @dancastellon mentioned, there is no more `/server/api` in release 1.14